src/services/datahub-client.ts

```diff
@@ -123,23 +132,37 @@ export class HttpDatahubClient implements DatahubClient {
   }: {
     domainName: number;
     dateFrom: Date;
-  }): Promise<{ name: string; totalVisitors: number; url: string }[]> {
-    const response = await this.customerGet<{
-      items: { page: string; visitorsQuantity: number }[];
-    }>(`domains/${domainName}/events/summarized-by-page`, {
-      startDate: dateFrom.toISOString(),
-      sortBy: 'visitors',
-    });
+  }): Promise<PageRankingResult> {
+    try {
+      const response = await this.customerGet<any>(
+        `domains/${domainName}/events/summarized-by-page`,
+        {
+          startDate: dateFrom.toISOString(),
+          sortBy: 'visitors',
+        },
+      );
 
-    // By the moment we are hard-coding it because DataHub does not have this
-    // information. I am looking you Leo :P
-    const urlSchema = 'http://';
+      // By the moment we are hard-coding it because DataHub does not have this
```

Comment:
I think this comment does not apply anymore right?

Reply:
we dont have the schema because datahub dont provide us, then need to use this for now :(

Reply:
Related ticket: [DBR-271](http://jira.makingsense.com/browse/DBR-271)